crates/cast/src/starknet_commands/account/deploy.rs

```diff
@@ -32,6 +36,7 @@ pub async fn deploy(
     network: &str,
     max_fee: FieldElement,
     wait: bool,
+    class_hash: Option<String>,
```

Comment:
Shouldn't we rather save `class_hash` to the `accounts-file` than require passing it twice?

Reply:
The file does not hold such information, and I'm not sure we should modify its structure by our own.
I suppose this is something that could be added to scarb.toml file instead?

Reply:
hmm, are you sure we can't change the file structure? In my opinion, it should keep all important account info and class_hash is the one

Reply:
I'm not 100% sure, but I know this file is also created and worked upon by cairo-lang etc, I don't want to take a risk and render it unusable for users that use different tools.

I agree it would be a nice place to put it, but this is somehting that has to be coordinated with starkware.

Reply:
Ok, so let's make an issue for this 🙏